public/apps/configuration/panels/role-mapping/external-identities-panel.test.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+import {
+  buildExternalIdentityState,
+  unbuildExternalIdentityState,
+  ExternalIdentitiesPanel,
+} from './external-identities-panel';
+import { shallow } from 'enzyme';
+import { ExternalIdentityStateClass } from './types';
+import React from 'react';
+import { EuiFieldText, EuiFlexGroup } from '@elastic/eui';
+import {
+  appendElementToArray,
+  removeElementFromArray,
+  updateElementInArrayHandler,
+} from '../../utils/array-state-utils';
+
+jest.mock('../../utils/array-state-utils', () => ({
+  appendElementToArray: jest.fn(),
+  removeElementFromArray: jest.fn(),
+  updateElementInArrayHandler: jest.fn().mockReturnValue(jest.fn()),
+}));
+
+describe('Role mapping - external identities panel', () => {
+  const externalIdentity1 = 'external_identity1';
+  it('buildExternalIdentityState', () => {
```

Comment:
Minor style: better to be consistent as a statement style instead of a variable name style.

Reply:
Agreed. Added statement.